### PR TITLE
Add Event.prototype.deepPath to w3c_event externs.

### PR DESCRIPTION
**Pull request by ak239**
This closes https://github.com/google/closure-compiler/issues/1561 .

[Review on Reviewable](https://reviewable.io/reviews/google/closure-compiler/1562)

### CODE REVIEW

File: externs/browser/w3c_event.js [hunk 1]
```diff
@@ -134,6 +134,12 @@ Event.prototype.timeStamp;
 Event.prototype.path;
 
 /**
+ * Present for events spawned in browsers that support shadow dom.
+ * @type {function():Array<!EventTarget>|undefined}
```
Comment (Dominator008) [inline externs/browser/w3c_event.js]: Please add a @see tag and include a URL to the section of the specs that describes this.

**Comment by ak239** [comment]
I signed it!

File: externs/browser/w3c_event.js (hunk 1 above)
Comment (ak239) [inline externs/browser/w3c_event.js]: Now, it's described in last editor version: http://w3c.github.io/webcomponents/spec/shadow/ . Can I add a link to it or I should wait when it'll become latest published version?

Comment (ak239) [inline externs/browser/w3c_event.js]: I've added link to sections in latest published version.

Comment (Dominator008) [inline externs/browser/w3c_event.js]: The draft link is fine

**Comment by Dominator008** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[externs/browser/w3c_event.js, line 138 \[r1\]](https://reviewable.io:443/reviews/google/closure-compiler/1562#-KApjTWx3dpHwtdcGAO0-r1-138:-KApzzUi2HvH-aSZMDpZ:-1116763747) ([raw file](https://github.com/google/closure-compiler/blob/133b56b25f51069b1948a22c454d97bb1275c88c/externs/browser/w3c_event.js#L138)):*
Can you make sure the typing is correct? Shouldn't it always return an Array?

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/google/closure-compiler/1562)*

**Comment by ak239** [comment]
*[externs/browser/w3c_event.js, line 138 \[r1\]](https://reviewable.io:443/reviews/google/closure-compiler/1562#-KApjTWx3dpHwtdcGAO0-r1-138:-KAq2GCavQCLpl5gaugR:893079530) ([raw file](https://github.com/google/closure-compiler/blob/133b56b25f51069b1948a22c454d97bb1275c88c/externs/browser/w3c_event.js#L138)):*
Spec said:
When invoked, it must return return a sequence consisting of event targets, that must be equivalent to processing the following steps.

If array is equivalent  to sequence then yes, always return an Array.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/google/closure-compiler/1562)*

**Comment by dimvar** [comment]
LGTM. Can you squash the commits? Then it's ready to merge.

**Comment by ak239** [comment]
I've squashed commits and pushed it.

**Pull request merged**